src/utility/src/base_decoder_libav.cpp

```diff
@@ -273,7 +274,7 @@ int BaseDecoderLibAV::addFilter(std::string_view filter)
 
     const std::string args =
         fmt::format("width={}:height={}:pix_fmt={}:time_base=1/1:sar={}/{}", m_videoDecCtx->width,
-                    m_videoDecCtx->height, m_videoDecCtx->pix_fmt,
+                    m_videoDecCtx->height, (int) m_videoDecCtx->pix_fmt,
```

Comment:
Why this change? Are you getting a compiler warning for providing an enum, or does it just format enums weirdly?

Reply:
I'm getting an error when compiling, because three is no formatter for the pix_fmt enum

Reply:
Ah interesting! It's possible that more recent versions are more picky about this (I assume that when we build, it's just converting the enum to the underlying type)

In any case, we prefer `int32_t` unless there's a compelling reason not to use it

Reply:
The reason I chose int is that pix_fmt is a "enum AVPixelFormat", and in the C language specification, the guarantee is that enum is the same size as int, so by casting to int, we'll be sure it will work on all platform. Although I guess any platform where int!=int32_t will probably break somewhere else.